modules/serverless-rn-1-4-0.adoc

```diff
@@ -0,0 +1,56 @@
+* {ServerlessProductName} has been updated to use Knative Serving Operator 0.11.1.
+* Knative Client is now available for download through the *Command Line Tools* page in the {product-title} web console.
+* The `KnativeServing` object's API group has changed in this release from `serving.knative.dev` to `operator.knative.dev`. You will need to adjust any of your scripts or applications that rely on the old API group to use the new API group. The {ServerlessProductName} installation instructions have been updated to use the new API group.
++
+When upgrading from {ServerlessProductName} 1.3.0 to 1.4.0, the {ServerlessOperatorName} will create a `KnativeServing` custom resource (CR) in the new API group for you. This CR will be a mirror of the `KnativeServing` CR in the old group that was used in {ServerlessProductName} 1.3.0.
++
+If you need to keep using the old group temporarily, you can use the old CR as before. However, this CR is deprecated and will eventually be removed.
++
+Once you have updated references to the new API group, you can remove any older CR versions and use the newly deployed `KnativeServing` CR instead. To safely do this without downtime, remove the owner reference from the newly deployed `KnativeServing` CR using:
++
+----
+ oc edit knativeserving.operator.knative.dev knative-serving -n knative-serving
+----
++
+After the owner reference has been removed, you can safely remove any older CR versions and start using the new one.
++
+[IMPORTANT]
+====
+If a previous version of the CR exists, changes to the new CR will be overwritten by the {ServerlessOperatorName}. While the old CR is still active, all changes need to be made to that CR.
+====
+
+[id="fixed-issues-1-4-0_{context}"]
+== Fixed issues
+* Connecting to a private, cluster local Knative Service from a namespace that was not part of the `knative-serving-ingress` Service Mesh was failing on `i/o timeout`. This issue is now fixed.
+
+* The `container_name` and `pod_name` metric labels were removed in {product-tile} 4.3. The documentation has been updated to use the new `container` and `pod` metric labels instead. If you are using metering with Serverless on {product-tile} 4.3 or later, you must update your Prometheus queries according to the current version of the Serverless metering documentation.
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+* Unqualified usage of `knativeserving` in `oc` commands no longer works because of the migration to a new API group. For example, this command will not work:
++
+----
+oc get knativeserving -n knative-serving
+----
++
+Use the explicit fully-qualified format instead. For example:
++
+----
+oc get knativeserving.operator.knative.dev -n knative-serving
+----
+
+* The first Knative Service created in a namespace can take over 10 seconds to become ready, waiting in `IngressNotConfigured` state. This is due to the time taken for the namespace to be added to the `ServiceMeshMemberRoll`.
```

Comment:
This has been known in 1.3.0 already. We even documented it in "Deploying a serverless application"

> The first time that a Knative service is created in a namespace, that  namespace will automatically receive a new networking configuration.  This might cause the initial service to take longer than is usually  required for a service to become ready.